source/whats-new.txt

```diff
@@ -57,8 +57,38 @@ The 3.0 driver release includes the following new features:
   This type is available in .NET 5 and later.
   To learn more about the ``Half`` type, see the
   `Half Struct <https://learn.microsoft.com/en-us/dotnet/api/system.half?view=net-8.0>`__
-  API reference page on MSDN. 
-
+  API reference page on MSDN.
+
+- Adds support for implicit client-side projection when using the ``Find()`` and
+  ``Select()`` methods. In previous versions of the driver, you could perform client-side
+  projection only after calling the ``ToEnumerable()`` or ``AsEnumerable()`` method, and
+  only by using the ``Select()`` method, similar to the following examples:
+
+  .. code-block:: csharp
+
+     var find = collection
+         .Find(doc => doc.Id == 1)
+         .ToEnumerable()
```

Comment:
This is not driver supported client-side projections. This is the user doing an EXPLICIT client-side projection by splitting their query into server-side and client-side parts and separating them by `ToEnumerable`.

Reply:
My understanding was that this is _currently_ the only way to do it, which is why I presented it as the "before" example. Is that wrong?

Either way, I rearranged a bit to hopefully make it clearer.